src/System.Collections.Immutable/tests/ImmutableListBuilderTest.cs

```diff
@@ -178,6 +178,17 @@ public void Remove()
         }
 
         [Fact]
+        public void RemoveAllBugTest()
+        {
+            var builder = ImmutableList.CreateBuilder<int>();
+            var elemsToRemove = new[]{0, 1, 2, 3, 4, 5}.ToImmutableHashSet();
+            foreach(var elem in new[]{0, 1, 2, 3, 4, 5, 6})
+                builder.Add(elem);
```

Comment:
Does this loops of Add calls do something different than just calling `builder.AddRange`? I'm curious why you chose to write a loop in the test.